src/main/java/frc/robot/subsystems/ball/feeder/FeederImpl.java

```diff
@@ -44,8 +55,10 @@ public void defineResources(){
     }
 
     public void unload() {
-        this.reverseFeeder();
-        // TODO implement sensor logic
+        while(!this.poller.checkUnloaded()){
+            this.reverseFeeder();
+        }
+        this.stop();
```

Comment:
Who are all the possible callers of `unload`? Is it safe to put a while-loop here, especially one without a `sleep` in it? If whatever calls this is going to do so periodically then the while-loop can be turned into a simple if-statement.

Reply:
Yeah that's true. I didn't think about that.

Reply:
AFAIK the controllers only call the method once (when you map the controls), so what's calling it isn't doing it periodically. I'll add a sleep to it for a few ms.